test/classic/classic_test.go

```diff
@@ -19,21 +19,22 @@ import (
 )
 
 const (
-	installSecretsPath = "../testdata/secrets/classic-fullstack-install.yaml"
+	installSecretsPath = "../testdata/secrets/install.yaml"
 )
 
 var testEnvironment env.Environment
 
 func TestMain(m *testing.M) {
 	testEnvironment = environment.Get()
+	testEnvironment.BeforeEachTest(namespace.DeleteIfExists(sampleapps.Namespace))
 	testEnvironment.BeforeEachTest(dynakube.DeleteIfExists())
 	testEnvironment.BeforeEachTest(oneagent.WaitForDaemonSetPodsDeletion())
-	testEnvironment.BeforeEachTest(namespace.DeleteIfExists(sampleapps.Namespace))
-	testEnvironment.BeforeEachTest(namespace.Recreate(dynakube.Namespace))
+	testEnvironment.BeforeEachTest(namespace.DeleteIfExists(dynakube.Namespace))
+	testEnvironment.BeforeEachTest(namespace.Create(dynakube.Namespace, nil))
```

Comment:
Nitpick: couldn't that be combined in a single `Recreate`?

Reply:
done